TopsortAnalytics/src/main/java/com/topsort/analytics/banners/BannerConfig.kt

```diff
@@ -0,0 +1,82 @@
+package com.topsort.analytics.banners
+
+/**
+ * Class that handles different type of Banner configurations
+ */
+sealed class BannerConfig private constructor() {
+
+    /**
+     * Banner configuration for landing page banners
+     *
+     * @property slotId id of the banner slot
+     * @property ids ids of the entities that are competing for the banner
+     * @property device can be "desktop" or "mobile"
+     * @property geoTargeting optional location for geo-targeted banners
+     */
+    data class LandingPage(
+        val slotId: String,
+        val ids: List<String>,
+        val device: String? = null,
```

Comment:
I think we should use an enum for device, what do you think @anonvt?

Reply:
I'd add that device should be set to `mobile` by default, as in most cases that will be the intended use case.

Reply:
you are right, I eas matching the optional from the api (which in turn is converted to desktop internally), but definitely, mobile should be the default here

Reply:
Agreed that an enum would be the best fit.